{
    "Logging": {
        "LogLevel": {
            "Default": "Information",
            "Microsoft.AspNetCore": "Warning"
        }
    },
    "EmailConfiguration": {
        "From": "afzalhossainjsr@gmail.com",
        "SmtpServer": "smtp.gmail.com",
        "Port": 587,
        "Username": "afzalhossainjsr@gmail.com",
        "Password": "jrkcorgwqafnefyu",

        "message_username": "Looks_Admin",
        "message_password": "Ta@Looks2022",
        "message_cli": "LOOKS",
        "message_countrycode": "880"

    },
    "AllowedHosts": "*",
    "ConnectionStrings": {
       // "DefaultConnection": "Server=103.222.21.142;Database=UserDB; user id=sa;password=@Afzal1234;Trusted_Connection=False; MultipleActiveResultSets=true;"
        "DefaultConnection": "Server=DESKTOP-3VQUQ5C;Database=UserDB;uid=sa;password=1234;Trusted_Connection=False; MultipleActiveResultSets=true;"

        //local https://stackoverflow.com/questions/17615260/the-certificate-chain-was-issued-by-an-authority-that-is-not-trusted-when-conn
        //"DefaultConnection": "Server=DESKTOP-3VQUQ5C;Database=UserDB; uid=sa;password=1234;Trusted_Connection=false;TrustServerCertificate=false;Encrypt=false;MultipleActiveResultSets=true;"

    },
    "JWT": {
        "ValidAudience": "User",
        "ValidIssuer": "http://localhost:5096",
        "Secret": "ThisIsSuperSecretKet"
    }
} 

